File: Gixor-Backend/Gixor-Api/Gixor-Api-GitHub/src/main/java/com/cpy3f2/Gixor/Domain/TrendyUser.java
package com.cpy3f2.Gixor.Domain;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author : simon
 * @description :
 * @last : 2024-11-05 11:01
 * Copyright (c) 2024. 保留所有权利。
 * @since : 2024-11-05 11:01
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
@JsonIgnoreProperties(ignoreUnknown = true)
public class TrendyUser {
    private String username;
    private String displayName;
    private String repoName;
    private String avatarUrl;
    private String description;
}

File: Gixor-Mobile/app/src/main/java/com/cpy3f2/gixor_mobile/model/entity/TrendyUser.kt
package com.cpy3f2.gixor_mobile.model.entity
import com.google.gson.annotations.SerializedName

/**
 * @author : sadSmile
 * @description :
 * @last : 2024-11-05 11:01
 * Copyright (c) 2024. 保留所有权利。
 * @since : 2024-11-05 11:01
 */
data class TrendyUser(
    @SerializedName("username")
    val username: String,
    @SerializedName("displayName")
    val displayName: String,
    @SerializedName("repoName")
    val repoName: String,
    @SerializedName("avatarUrl")
    val avatarUrl: String,
    @SerializedName("description")
    val description: String
)
